sentry_streams/tests/pipeline/test_msg_codecs.py

```diff
@@ -78,3 +88,136 @@ def test_msg_no_found_schema() -> None:
     with pytest.raises(ValueError) as e:
         _get_codec_from_msg(msg)
     assert "Kafka topic invalid-schema has no associated schema" in str(e.value)
+
+
+def test_parquet_serializer_with_polars_schema() -> None:
+    import polars as pl
+
+    msg: Message[Sequence[Mapping[Any, Any]]] = PyMessage(
+        payload=[
+            {
+                "org_id": 420,
+                "project_id": 420,
+                "name": "s:sessions/user@none",
+                "tags": {
+                    "sdk": "raven-node/2.6.3",
+                    "environment": "production",
+                    "release": "sentry-test@1.0.0",
+                },
+                "timestamp": 11111111111,
+                "type": "s",
+                "retention_days": 90,
+                "value": [1617781333],
+            },
+            {
+                "org_id": 420,
+                "project_id": 420,
+                "name": "s:sessions/user@none",
+                "tags": {
+                    "sdk": "raven-node/2.6.3",
+                    "environment": "production",
+                    "release": "sentry-test@1.0.0",
+                },
+                "timestamp": 11111111111,
+                "type": "s",
+                "retention_days": 90,
+                "value": [1617781333],
+            },
+        ],
+        schema="example-schema",
+        headers=[],
+        timestamp=0.0,
+    )
+
+    schema = {
+        "org_id": Int64(),
+        "project_id": Int64(),
+        "name": String(),
+        "tags": Struct(
+            [
+                Field("sdk", String()),
+                Field("environment", String()),
+                Field("release", String()),
+            ]
+        ),
+        "timestamp": Int64(),
+        "type": String(),
+        "retention_days": Int64(),
+        "value": List(Int64()),
+    }
+
+    result = parquet_serializer(msg, schema)
+
+    assert isinstance(result, bytes)
+
+    df = pl.read_parquet(BytesIO(result))
+    assert df.shape == (2, 8)
+
+    expected_df = pl.DataFrame(
+        [
```

Comment:
If this is identical to the one above, please put the list in a constant and use it both to build the parquet dataframe and to verify that, otherwise we will eventually update one forgetting the other.